app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
ah yes this was a tricky naming situation -- "Reverse" is not as clear to me. Perhaps "Default' although that feels similar to "Base", too. Now that I know we used to call them "Multi" I always call it that in my head....ack idk! what do you think?